src/Detector/Python/PythonDetector.cs

```diff
@@ -40,15 +40,36 @@ public PlatformDetectorResult Detect(DetectorContext context)
             var appDirectory = string.Empty;
             var hasRequirementsTxtFile = false;
             var hasPyprojectTomlFile = false;
+
             if (sourceRepo.FileExists(PythonConstants.RequirementsFileName))
             {
                 _logger.LogInformation($"Found {PythonConstants.RequirementsFileName} at the root of the repo.");
                 hasRequirementsTxtFile = true;
+
+                // Warning if missing django module
+                bool hasDjangoModule = false;
+                string filePath = $"{sourceRepo.RootPath}/{PythonConstants.RequirementsFileName}";
+                using (var reader = new StreamReader(filePath))
+                {
+                    while (!reader.EndOfStream && !hasDjangoModule)
+                    { 
+                        string line = reader.ReadLine().ToLower();
+                        if (line.StartsWith("django"))
+                        {
+                            hasDjangoModule = true;
+                        }
+                    }
+                }
+                if (!hasDjangoModule)
+                {
+                    _logger.LogWarning($"Missing django module in {PythonConstants.RequirementsFileName}");
+                }
             }
             else
             {
-                _logger.LogInformation(
-                    $"Cound not find {PythonConstants.RequirementsFileName} at the root of the repo.");
+                string errorMsg = $"Cound not find {PythonConstants.RequirementsFileName} at the root of the repo.";
```

Comment:
same comment about this error message

Reply:
Per our discussion, we'll be creating a separate method for logging. More updates to come

Reply:
First commit of improved logging: https://github.com/microsoft/Oryx/pull/1099/commits/b25ef613569a3d4d40e0b1ece80be124903df65d